ghost/i18n/locales/pl/portal.json

```diff
@@ -97,33 +97,33 @@
     "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Jeśli wykonałeś wszystkie te czynności i nadal nie otrzymujesz wiadomości, możesz skontaktować się z pomocą techniczną pod tym adresem {{supportAddress}}.",
     "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "W przypadku trwałych problemów podczas wysłania newslettera, wiadomości email zostaną wyłączone na koncie.",
     "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "W kliencie poczty dodaj {{senderEmail}} do listy kontaktów. Sygnalizuje to dostawcę poczty, że wiadomości wysyłane z tego adresu powinny być zweryfikowane.",
-    "Invalid email address": "",
-    "Jamie Larson": "",
-    "jamie@example.com": "",
+    "Invalid email address": "Nieprawidłowy adres e-mail",
+    "Jamie Larson": "Jamie Larson",
```

Comment:
"Jamie Larson" can be replaced with a common Polish name.  It appears as a placeholder in the name field on sign-up.